### PR TITLE
#160447638 Analytics for total monthly meetings duration

### PR DESCRIPTION
**Pull request by Bonifase**
#### What does this PR do?
 Enable admin to know the total meeting duration for each room within a specific month.
#### How should this be manually tested?
- Clone this branch in your local machine, Run the server
- Test the queries at localhost:5000/mrm
- Run `monthlyDurationsOfMeetings` query with `locationId`, `month` and `year` as the keyword arguments as shown in the screen shot below.
#### What are the relevant pivotal tracker stories?
[#160447638](https://www.pivotaltracker.com/story/show/160447638)
#### Screenshots 
     When an admin runs the query with locationId, year and month.
screen shot 2018-10-03 at 15 57 31

### CODE REVIEW

File: tests/test_rooms/test_analytics_total_daily_durations.py [hunk 1]
```diff
@@ -17,4 +17,5 @@ def test_total_daily_durations(self):
         headers = {"Authorization": "Bearer" + " " + admin_api_token}
         response = self.app_test.post('/mrm?query='+get_daily_meetings_total_duration_query, headers=headers)  # noqa: E501
         actual_response = json.loads(response.data)
+        print("This is a response", actual_response)
```
Comment (3V0L) [inline tests/test_rooms/test_analytics_total_daily_durations.py]: I think you should get rid of this print statement and we will be good to go

File: tests/test_rooms/test_monthly_meetings_duration.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+from fixtures.token.token_fixture import admin_api_token
+from fixtures.room.room_monthly_meeting_duration_fixtures import (
+    get_monthly_meetings_total_duration_query,
+    get_monthly_meetings_total_duration_response
+)
+
+
+class TotalMonthlyDurations(BaseTestCase):
+
+    def test_total_monthly_meeting_durations(self):
```
Comment (shemogumbe) [inline tests/test_rooms/test_monthly_meetings_duration.py]: You would now need to alter your tests and fixtures to conform to the new  format of tests and fixtures in order to remove duplicates

Comment (Bonifase) [reply tests/test_rooms/test_monthly_meetings_duration.py]: Ok. Let me put that into consideration.
